udata/templates/dataset/resource/list-item.html

```diff
@@ -1,6 +1,7 @@
 {% set resource_format = resource.format|trim|lower or 'data' %}
 <div id="resource-{{resource.id}}" class="list-group-item"
     data-checkurl="{{ url_for('api.check_dataset_resource', dataset=dataset.id, rid=resource.id) }}"
+    data-extras="{{ resource.extras|to_json }}"
```

Comment:
Maybe reuse the `json-ld` already used in this page to serialize structured data (data and its extras, ressources...).
See:
- https://github.com/opendatateam/udata/blob/master/udata/core/dataset/models.py#L525-L544
- https://github.com/opendatateam/udata/blob/master/udata/core/dataset/models.py#L219-L259

Reply:
👍